src/Esprima/JavascriptParser.cs

```diff
@@ -208,10 +208,13 @@ private protected void CollectComments()
             {
                 var e = comments[i];
 
-                e.Value = _scanner.Source.Slice(e.Slice.Start, e.Slice.End);
-                e.Location = new Location(e.Location.Start, e.Location.End, _errorHandler.Source);
+                var comment = new SyntaxComment(e.Type, value: _scanner.Source.Slice(e.Slice.Start, e.Slice.End))
+                {
+                    Range = new Range(e.Start, e.End),
+                    Location = new Location(in e.StartPosition, in e.EndPosition, _errorHandler.Source)
```

Comment:
nit: `in` modifier on call site probably redundant

Reply:
True but I like to be explicit about this.